File: C_Programming/Practice Program/program112.c
#include<stdio.h>

void Display(int Arr[])

{
    int icnt = 0;
    for(icnt = 0 ; icnt <8 ;icnt++)
    {
        printf(" %d\n",Arr[icnt]);
    }    
}
 
int main()
{
    int Brr[8] = {10,20,30,40,50,60,70,80};

    Display(Brr);

    return 0;
}

File: Java_Programming/Competitive Programming/program112.java
 /*
    Write a Java program to extend the previous assignment to distinguish
    between files and directories and display the size of files.
*/

import java.io.*;
import java.util.*;

class program112
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter directory:");
        String str = sobj.nextLine();

        File fobj = new File(str);

        if(fobj.exists())
        {
            if(fobj.isDirectory())
            {
                File Arr[] = fobj.listFiles();

                for(int i = 0; i < Arr.length; i++)
                {
                    if(Arr[i].isFile())
                    {
                        System.out.println("[FILE] " + Arr[i].getName()
                                + "    " + Arr[i].length() + " bytes");
                    }
                    else if(Arr[i].isDirectory())
                    {
                        System.out.println("[DIR] " + Arr[i].getName());
                    }
                }
            }
            else
            {
                System.out.println("It is not a directory");
            }
        }
        else
        {
            System.out.println("Directory does not exist");
        }
    }
}
